app/src/main/java/com/nervousfish/nervousfish/data_objects/DatabasePass.java

```diff
@@ -0,0 +1,76 @@
+package com.nervousfish.nervousfish.data_objects;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+import com.nervousfish.nervousfish.modules.cryptography.EncryptedSaver;
+
+import java.io.InvalidObjectException;
+import java.io.ObjectInputStream;
+import java.io.Serializable;
+
+
+/**
+ * A databasePass object to save the enccryption elements
```

Comment:
Can you elaborate a bit more on this? I have no idea what this class is for. Maybe you should also consider renaming the class, because we don't use abbreviations anywhere in the project (assuming that Pass is the abbreviation of something)